Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/combat_correspondent.yml

```diff
@@ -0,0 +1,72 @@
+- type: entity
+  parent: ColMarTechBase
+  id: ColMarTechCrewCombatCorrespondent
+  name: ColMarTech Combat Correspondent Equipment Rack
+  description: An automated rack hooked up to a colossal storage of Reporter standard-issue equipment.
+  components:
+  - type: AccessReader
+    access:
+    - [ "CMAccessPress" ]
+  - type: Sprite
+    sprite: _RMC14/Structures/Machines/VendingMachines/ColMarTech/clothing.rsi
+  - type: CMAutomatedVendor
+    jobs:
+    - CMCorrespondent
+    sections:
+    - name: Standard Equipment
+      takeAll: CMStandard
+      entries:
+      - id: RMCVendorBundleCrewCombatCorrespondent
+      # Portable Press Fax Machine - /obj/item/device/fax_backpack
+      - id: RMCCameraBroadcasting
+      - id: CMSatchel
+    - name: Uniform
+      choices: { CMUniform: 1 }
+      entries:
+      - id: RMCJumpsuitCorrespondentBlack
+        name: black uniform
+      - id: RMCJumpsuitCorrespondentOrange
+        name: orange uniform
+      - id: RMCJumpsuitCorrespondentRed
+        name: red uniform
+    - name: Armor
+      choices: { CMArmor: 1 }
+      entries:
+      - id: CMArmorPress
+      - id: RMCCoatCorrespondentBlue
+        name: blue vest
+      # black vest
+      - id: RMCCoatCorrespondentBlack
+        name: black coat
+      - id: RMCCoatCorrespondentGreen
+        name: green coat
+    - name: Helmet
+      choices: { CMHelmet: 1 }
+      entries:
+      - id: CMArmorHelmetPress
+    - name: Refills
+      entries:
+      # /obj/item/device/camera 10
+      # /obj/item/device/camera_film 5
+      # /obj/item/device/toner 5
+      # /obj/item/storage/box/tapes 15
+      - id: CMPaper
+        amount: 30
+      - id: CMHandLabeler
+        amount: 3
```

Comment:
This is supposed to be 10 cameras, 5 camera films, 5 toners, 15 regulation tapes (none of those we have I think), 10 paper bins

Reply:
Kadeo64 is going to need to settle for no CMHandLabelers, there's some in medical and Req so we don't have to AHelp him for trying to use it to bypass the fix for xenos being able to read papers